MVP Module/Source/Feed UI/Composers/FeedLoaderPresentationAdapter.swift

```diff
@@ -21,7 +22,7 @@ final class FeedLoaderPresentationAdapter: FeedViewControllerDelegate {
 				self?.presenter?.didFinishLoadingFeed(with: feed)
 
 			case let .failure(error):
-				self?.presenter?.didFinishLoadingFeed(with: error)
+				self?.errorPresenter?.didFinishLoadingFeed(with: error)
```

Comment:
The `FeedLoaderPresentationAdapter` is responsible for adapting a single `FeedLoader` `load` method (and its possible results) and notify the presenter. There's no need to have another presenter for errors.
```suggestion
				self?.presenter?.didFinishLoadingFeed(with: error)
```

Reply:
resolved